pkg/auth/keymanager.go

```diff
@@ -4,7 +4,7 @@ import (
 	"bytes"
 	"crypto/rsa"
 	"encoding/json"
-	"errors"
+	stderrors "errors"
```

Comment:
why? I would rather use just `errors`.

Reply:
ah, that's because I used the same alias in another file, and since the repo has its own `errors` package, I wanted to make sure that the standard `errors` package is aliased everywhere (for the sake of consistency). Plus, there's also the `k8s.io/apimachinery/pkg/api/errors` package which I aliased to `apierrors`.

Reply:
so what about `errs` as is used in other repos?

Reply:
`errs` is usually `github.com/pkg/errors` (AFAIR)

Reply:
my point is - I'm not aware of any single file where we would use `stderrors` as an alias for `errors` import. we usually create custom aliases for the other error imports not this standard one. I thought that it would be fine to keep it consistent with other files/repos.
I just shared my preference, but I don't wan to block this PR because of the import name (which is really a detail).

Reply:
fair point. I noticed the usage of "crterrors" in other files of this repo, so I will use that instead, and keep "errors" as-is

Reply:
see ccbb84a